test/common/upstream/health_check_fuzz_test.cc

```diff
@@ -15,6 +15,12 @@ DEFINE_PROTO_FUZZER(const test::common::upstream::HealthCheckTestCase input) {
     return;
   }
 
+  if (input.health_check_config().health_checker_case() ==
+      envoy::config::core::v3::HealthCheck::kCustomHealthCheck) {
+    ENVOY_LOG_MISC(trace, "Fuzz engine created Custom Health Checker");
```

Comment:
Maybe the log should be rephrased - "Custom Health Checker currently unsupported, skipping".

Reply:
Changed.